pvlib/tracking.py

```diff
@@ -43,24 +43,42 @@ class SingleAxisTracker(PVSystem):
         between the tracking axes has a gcr of 2/6=0.333. If gcr is not
         provided, a gcr of 2/7 is default. gcr must be <=1.
 
+    cross_axis_tilt : float, default 0.0
+        The angle, relative to horizontal, of the line formed by the
+        intersection between the slope containing the tracker axes and a plane
+        perpendicular to the tracker axes. The slope is assumed to be a plane
+        containing all tracker axes. Cross-axis tilt follows the same
+        right-handed convention as tracker rotation. For example a tracker with
+        axis azimuth facing south will have a negative cross-axis tilt if
+        sloped down to the east and positive if sloped up. Use
+        :func:`~pvlib.tracking.calc_cross_axis_tilt` for more complicated
+        slopes. [degrees]
```

Comment:
```suggestion
        The angle, relative to horizontal, of the line formed by the
        intersection between the plane containing the tracker axes and a plane
        perpendicular to the tracker axes. Cross-axis tilt angle should be
        specified using a right-handed convention. For example, for a tracker with
        axis azimuth at 180 degrees (facing south), the cross-axis tilt angle is the
        angle from horizontal of the tracker axis plane to the east, which will be
        negative if the tracker axis plane slopes down to the east and positive if
        the tracker axis plane slopes up to the east. Use
         :py:func:`~pvlib.tracking.calc_cross_axis_tilt` to calculate `cross_axis_tilt`. [degrees]
```

Reply:
I think we may need an image to explain this without confusion. I'm not convinced that we will find the perfect wording or that more words will make this any less abstract. @kanderso-nrel okay to use this image?
![image](https://user-images.githubusercontent.com/1385621/92280815-1ce9e700-eeaf-11ea-996f-46882fdf3dec.png) I think it captures everything except &beta;<sub>c</sub> which is just the angle between _**v**_ and _**tx**_

Reply:
Is it possible to include an image in the docstrings?

Reply:
Yeah, OK to use any image from the report.  And I would expect sphinx to be able to handle images in the docstring RST, but I wonder if a gallery example would be better for that level of detail?  I would venture that most people wanting to use `singleaxis` don't care about sloped terrain.  Just throwing the idea out there.

Reply:
OK, I agree with @kanderso-nrel , I think most users won't set cross-axis tilt or care about a tracker-axes slope, so I also agree a gallery tutorial would be a more suitable and impactful location for images. Sorry to bother you but would you be willing or interested in doing that? Then I believe the gallery thumbnail and link will magically appear on the doc pages right? thanks!

Reply:
And I sense we aren't aligned on the terms. I think the concept is clear: the tracker axes are co-planar, and that plane is (uniquely) described by the azimuth of it's normal projected to horizontal (CD), and an inclination angle of the normal from vertical (Delta gamma in the figure above). The plane itself is what I've called tracker axis plane, and what @mikofski calls the slope (which I think is vague). 

For `axis_tilt` and `cross_axis_tilt`, place a ray in the tracker axis plane in the direction of `axis_azimuth` (AB). The angle from horizontal of the `axis_azimuth` ray is `axis_tilt` (beta_alpha). The angle from horizontal of a ray 90 degrees counter-clockwise from the `axis_azimuth` ray is `cross_axis_tilt` (not shown, but would be the angle from horizontal of v).

There is another small change from the previous `singleaxis` that was throwing me off: in the previous version, `axis_azimuth` was the y-coordinate of the tracker frame of reference. Here, its in the x- direction. Not a wrong change, but a change in how I need to visualize this new geometry. I probably need to read the unedited sections of comments again.

Reply:
>The plane itself is what I've called tracker axes plane, and what @mikofski calls the slope (which I think is vague).

I agree it's vague, that's why I preferred the "system plane" referring the the entire PV system of all tracker axes. I thought _you_ preferred "slope" but I'm happy that we're all moving toward "tracker axes plane" - I've changed most references now to say the "slope of the tracker axes plane" or "slope containing tracker axes" or just the "tracker axes plane". My only problem with "tracker axes plane" is IMO this is much more easily confused with "tracker axis" (singular) than "system" (which to me implies the plurality of tracker axes).

>in the previous version, `axis_azimuth` was the y-coordinate of the tracker frame of reference. Here, its in the x-direction.

There's no change. The `axis_azimuth` indicated by <it>t<sub>y</sub><it> is still the still the y-axis in the tracker coordinate system. The x-axis in the tracker coord-sys is perpendicular to the y-axis (the tracker axis) and in the horizontal plane.

Reply:
tracker coordinates _t_ in red and global in blue
![image](https://user-images.githubusercontent.com/1385621/92287778-9e497580-eebf-11ea-86b3-02bc42eaf984.png)

tracker rotation &theta;
![image](https://user-images.githubusercontent.com/1385621/92287800-ae615500-eebf-11ea-9d35-831007dab364.png)

cross-axis tilt, &beta;<sub>c</sub>:
![image](https://user-images.githubusercontent.com/1385621/92287815-bfaa6180-eebf-11ea-9a2c-b261629247ac.png)